app/services/attribute_asserter.rb

```diff
@@ -44,9 +44,11 @@ def default_attrs
   end
 
   def add_bundle(attrs)
+    use_ascii = bundle.include?(:ascii)
```

Comment:
Why does `ascii?` need to be memoized? `bundle` is already memoized, and all `ascii?` would be doing is checking if the bundle includes the symbol `:ascii`, which is not an expensive operation, and it's only called once, right?